### PR TITLE
Added support for writing pcaps with nanosecond timestamp

### PR DESCRIPTION
**Pull request by rkulseth**
This will allow a user to write pcaps with nanosecond precision timestamps without having to use pcapNG format.  The change is 100% backwards compatible.

### CODE REVIEW

**Comment by rkulseth** [comment]
I signed it.

File: pcapgo/write.go [hunk 1]
```diff
@@ -50,14 +72,20 @@ const versionMinor = 4
 //  w2.WritePacket(gopacket.CaptureInfo{...}, data2)
 //  f2.Close()
 func NewWriter(w io.Writer) *Writer {
-	return &Writer{w: w}
+	return &Writer{w: w, tsScaler: nanosPerMicro}
 }
 
 // WriteFileHeader writes a file header out to the writer.
-// This must be called exactly once per output.
+// This must be called exactly once per output.  set usenanos to
```
Comment (notti) [inline pcapgo/write.go]: Looks like the `usenanos` remark is from an older version of the patch. Looks like this can be left out

Comment (rkulseth) [reply pcapgo/write.go]: Yep, that's exactly what happened.  Good catch.  Thank You.

**Comment by notti** [comment]
Thanks for the contribution.

**Pull request merged**